plugins/com.redhat.fabric8analytics.lsp.eclipse.ui/build.properties

```diff
@@ -6,6 +6,7 @@ bin.includes = META-INF/,\
                plugin.properties,\
                server/fabric8-analytics-lsp-server-master/,\
                templates/index.html,\
-               templates/
+               templates/,\
+               scripts/
```

Comment:
On the other hand, scripts folder does not need to be included

Reply:
I think we need scripts in build.properties as it contains the bashscript that would be running 
 inorder to download ca-lsp-server.

Reply:
The build.properties defines what should be packaged into Eclipse plugin - the script is needed only during build not during runtime in Eclipse